markdown/events/2022/bytesize-37-gathertown.md

```diff
@@ -24,6 +24,65 @@ It is our hope that these talks / videos will build an archive of training mater
+
+[10:22](https://www.youtube.com/watch?v=nRY1_FvL7Pk&t=622)
+On the right hand side, we have the second... the one on the main platform is for the speakers. If you ever have any questions, you can go and line up over here, exactly, nice queue. Whoever's standing on this orange bit will also be projected the room. if you have any questions during the talk, you can come up here to ask your questions. Now, the final room we can go into is where we have all the social activities, this is here in the nf-core cafe. Here's a good example of the private space around all the tables. We'll make sure this is everywhere. Anyone not in this square of the green square there cannot hear us. But if Maxime comes in, now he comes in, he can now join the conversation. And so, you have these private conversations as you want.
+
+[11:15](https://www.youtube.com/watch?v=nRY1_FvL7Pk&t=675)
+Other things about the user interface, there is also text chat if necessary. You can see this here. If you ever need to refer to the code of conduct, that's at the top here. You can change between different ways of chatting with people, either private chat to everybody in the entire space, like the entire world, and also any people in your close proximity. Another thing you can do is let's say you are lost. There's a little mini map down here, which can tell you where you are, and it'll point out where anybody else is. particularly in bigger spaces, if you can't find someone, you can look here. You can also search for people or follow people if necessary. Let's say, you know, I want to meet Matthias, I can press follow here. Automatically, my little avatar will go and walk towards him in the most fast route possible, then we can run. Also I can keep following him if he wants to lead me somewhere. If you're a bit too nervous and get stuck or lost, you can just follow people, that speeds things up as well. If at any point you get stuck, you can also... let's say, I come here, I can't walk through Matthias, but if I need to walk through Matthias because it's too big of a crowd, if I hold down G, I can then walk through the person. I turn them into a ghost, so G for ghost, and you can get through them. If you ever get stuck, that normally will help you with this.
```

Comment:
```suggestion
Other things about the user interface, there is also text chat if necessary. You can see this here. If you ever need to refer to the code of conduct, that's at the top here. You can change between different ways of chatting with people, either private chat to everybody in the entire space, like the entire world and also any people in your close proximity. Another thing you can do is let's say you are lost. There's a little mini-map down here, which can tell you where you are, and it'll point out where anybody else is. particularly in bigger spaces, if you can't find someone, you can look here. You can also search for people or follow people if necessary. Let's say, you know, I want to meet Matthias, I can press follow here. Automatically, my little avatar will go and walk towards him in the fastest route possible, then we can run. Also, I can keep following him if he wants to lead me somewhere. If you're a bit too nervous and get stuck or lost, you can just follow people, that speeds things up as well. If at any point you get stuck, you can also... let's say, I come here, I can't walk through Matthias, but if I need to walk through Matthias because it's too big of a crowd, if I hold down G, I can then walk through the person. I turn them into a ghost, so G for ghost, and you can get through them. If you ever get stuck, that normally will help you with this.
```